background/services/chain/index.ts

```diff
@@ -133,6 +139,11 @@ export type QueuedTxToRetrieve = {
   hash: HexString
   firstSeen: UNIXTime
 }
+/**
+ * The first element in the array is the transaction info the second is the priority.
+ * The priority value is a number in the range 0-1. Where 0 value is the highest priority.
+ */
+export type PriorityQueuedTxToRetrieve = [QueuedTxToRetrieve, number]
```

Comment:
Idea: maybe instead of an array, it would be easier to use an object to eliminate index-based references.
```suggestion
export type PriorityQueuedTxToRetrieve = { transaction: QueuedTxToRetrieve, priority: number }
```